pages/finished.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from "react";
+import { Books } from 'components/Books'
+import * as Api from 'services/api'
+import { IBook, IBooks } from 'components/Books/props'
+
+export const Finished = () => {
+  const [books, setBooks] = useState<IBooks[]>([]);
+
+  const loadData = () => {
+    Api.getFinishedBooks().then((response) => {
+      const data = response.data;
+      setBooks(data);
+    });
+  }
+
+  const markAsUnreadBook = async (bookId: number, onComplete: (response: IBook) => void) => {
+    Api.MarkAsUnread(bookId).then((response) => {
```

Comment:
this function has `async` but you are using `then`. Feel free to use `await` instead.

Also - I don't remember but will `fetch` raise an error if the `response` is not ok? If not, then you won't need to wrap this in a `try / catch` function.